CMakeLists.txt

```diff
@@ -615,4 +627,29 @@ else ()
   endif ()
 endif (IS_64BIT_BUILD EQUAL 1)
 
+
+# The prebuilt IDL headers aren't suitable for 32 bit builds, attempt to use the ones 
+# generated when building corguids by including them before the prebuilt ones
+if (WIN32 AND CLR_CMAKE_PLATFORM_ARCH_I386)
```

Comment:
We have `CLR_CMAKE_PLATFORM_UNIX_TARGET_AMD64` perhaps it'd be more consistent to have `CLR_CMAKE_PLATFORM_WINDOWS_TARGET_AMD64` etc?